pkg/artifactory/resource/security/resource_artifactory_api_key.go

```diff
@@ -32,6 +32,10 @@ func ResourceArtifactoryApiKey() *schema.Resource {
 				Type:      schema.TypeString,
 				Computed:  true,
 				Sensitive: true,
+				Deprecated: "An upcoming version will support the option to block the usage/creation of API Keys (for admins to set on their platform).\n" +
+					"In September 2022, the option to block the usage/creation of API Keys will be enabled by default, with the option for admins to change it back to enable API Keys.\n" +
+					"In January 2023, API Keys will be deprecated all together and the option to use them will no longer be available.\n" +
+					"Please check the release notes: https://www.jfrog.com/confluence/display/JFROG/Artifactory+Release+Notes#ArtifactoryReleaseNotes-Artifactory7.38.4",
```

Comment:
This should be set at the resource schema top level attribute `DeprecationMessage` instead. See https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema#Resource

Reply:
yup, moved.